tests/python/gpu/test_operator_gpu.py

```diff
@@ -1918,6 +1918,65 @@ def test_softmax_activation():
         assert_almost_equal(cpu_a.grad.asnumpy(), gpu_a.grad.asnumpy(),
                 atol = 1e-3, rtol = 1e-3)
 
+
+@with_seed()
+def test_bilinear_sampler_versions():
```

Comment:
why is this test function being added to test_operator_gpu.py when the old one was in test_operator.py? Do we have a similar test function just for CPU too, or do we now need to run GPU tests to test the CPU implementation for this case?

Reply:
yeah since we're checking the consistency of multiple versions, you need to run it where every version of it is available.